changelog.txt

```diff
@@ -13,6 +13,7 @@ V.Next
 - [MINOR] Adds new API to support ADAL/MSAL migration off of AuthenticationSettings#setSecretKey to a 'managed key' generated by Common (#1244)
 - [PATCH] Fix MSAL-CPP injecting javascript loop (#1238)
 - [MINOR] Device PoP keys are now generated with attestation flags and expose a certificate chain getter (#1247)
+- [MINOR] Added logging functionality to track the request status.(#1237)
```

Comment:
(As it stands this change is actually MAJOR (due to API breaks in `CommandResult`), if you make the ctor changes additive this PR will become a MINOR chage)

Reply:
made the ctor changes additive to make it MINOR